lib/diameter-connection.js

```diff
@@ -75,6 +74,11 @@ function DiameterConnection(options, socket) {
                         }
                     }
                     buffer = buffer.slice(messageLength);
+                } else {
+                    // Header has been collected in the buffer, but not the entire message.
+                    // So, end this event handler, and wait for another data event, with the
+                    // rest of the message. 
+                    return;
```

Comment:
Might be clearer to just `break` instead of `return` since the function doesn't do anything else afterward.

Reply:
Agree